src/app/admin/admin-reports/filtered-items/filtered-items.component.html

```diff
@@ -0,0 +1,156 @@
+                        </ng-template>
+                        <ng-template ngbPanelContent>
+                            <select id="collSel" name="collSel" multiple="multiple" size="10" formControlName="collections">
+                                <option *ngFor="let item of collections" [value]="item.id" [disabled]="item.disabled">{{item.name$ | async}}</option>
+                            </select>
+                            <div>
+                                <button class="query-button" (click)="submit()">{{'admin.reports.items.run' | translate}}</button>
+                            </div>
+                        </ng-template>
+                    </ngb-panel>
+                    <ngb-panel id="metadataFieldQueries">
+                        <ng-template ngbPanelTitle>
+                            {{'admin.reports.items.section.metadataFieldQueries' | translate}}
+                        </ng-template>
+                        <ng-template ngbPanelContent>
+                            <fieldset id="predefqueries">
+                                <label>
+                                    {{'admin.reports.items.predefinedQueries' | translate}}
+                                </label>
+                                <select id="predefselect" formControlName="presetQuery" (change)="setPresetQuery()">
+                                    <option *ngFor="let item of presetQueries" [value]="item.id" [selected]="item.isDefault">{{item.label | translate}}</option>
+                                </select>
+                            </fieldset>
+                            <div id="queries">
+                                <div class="metadata" *ngFor="let pred of queryPredicatesArray().controls; let i = index">
+                                    <div [formGroup]="pred">
+                                    <select class="query-tool" formControlName="field" class="predicate">
+                                        <option *ngFor="let item of metadataFieldsWithAny" [value]="item.id">{{item.name$ | async}}</option>
+                                    </select>
+                                    <span>&nbsp;</span>
+                                    <select class="query-tool" formControlName="operator" class="predicate">
+                                        <option *ngFor="let item of predicates" [value]="item.id">{{item.name$ | async | translate}}</option>
+                                    </select>
+                                    <span>&nbsp;</span>
+                                    <input class="predicate" formControlName="value"/>
+                                    <span>&nbsp;</span>
+                                    <button class="predicate" (click)="addQueryPredicate()">+</button>
+                                    <button class="predicate" [disabled]="deleteQueryPredicateDisabled()" (click)="deleteQueryPredicate(i)">–</button>
+                                    </div>
```

Comment:
Here, and in all other places where you render form fields or buttons, please use [bootstrap form classes](https://getbootstrap.com/docs/4.6/components/forms/) That way forms will look consistent across the repository, and will be easier to theme.

Reply:
W will try with the classes from the link you provided.

Reply:
Done.